### DISCUSSION TITLE
Improving UX of query viewer

### DISCUSSION THREAD

**Discussion by mgralikowski** [spatie/ray#954] [Ideas]
I am trying Ray instead of Debugbar I used to use for a long time. The most important function is to easily preview all queries run during a request (web and API - where Ray wins) with an execution time and duplicate occurs. 

To be honest UX compared to debugbar is poor, every request generates a lot of queries so as a result I have a lot of scrolling over not very readable queries. In the Debugbar the list contains rows with readable one-liners so it is easy to find interesting us queries. In the Ray even collapsing all queries is not helpful.
![image](https://github.com/user-attachments/assets/6f9cccbb-05b4-4724-9b33-cbc20af82a28)

![image](https://github.com/user-attachments/assets/d5eb66e3-2e36-4f44-b013-40861cc674f8)

Please, improve it a bit. The minimum is to make more readable collapsed sections and make its default behavior (configurable from the APP or config).

**Comment by mambax7** [comment]
Totally agree! See my post about the GUI from  [CodeSite for Delphi](https://raize.com/codesite/):
https://github.com/spatie/ray/discussions/1017
